Source/WebGPU/WebGPU.xcodeproj/project.pbxproj

```diff
@@ -986,6 +1085,51 @@
 			};
 			name = Production;
 		};
+		97FA1A8329C085740052D650 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 97FA1AA229C0BB700052D650 /* wgslc.xcconfig */;
+			buildSettings = {
+			};
+			name = Debug;
+		};
+		97FA1A8429C085740052D650 /* Release */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 97FA1AA229C0BB700052D650 /* wgslc.xcconfig */;
+			buildSettings = {
+			};
+			name = Release;
+		};
+		97FA1A8529C085740052D650 /* Production */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 97FA1AA229C0BB700052D650 /* wgslc.xcconfig */;
+			buildSettings = {
+			};
+			name = Production;
+		};
+		97FA1A9129C086CE0052D650 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CODE_SIGN_STYLE = Automatic;
+				PRODUCT_NAME = "$(TARGET_NAME)";
```

Comment:
Please delete these

Reply:
What are "these", the two entries under `buildSettings`? This was generated by Xcode when I created the target, and I'm not entirely sure what to keep or remove.

Reply:
Yes, those 2 entries. Any build configurations should be in .xcconfig files, not here.